apps/frontend/src/app/5_pages/MarketMakingPage/components/AdjustAndDepositModal/components/NewPoolStatistics/NewPoolStatistics.tsx

```diff
@@ -55,37 +120,53 @@ export const NewPoolStatistics: FC<NewPoolStatisticsProps> = ({
           amount.isZero() ? (
             t(translations.common.na)
           ) : (
-            <AmountRenderer value={newPoolBalance} suffix={token} />
+            <AmountRenderer
+              value={fromWei(newPoolBalanceA.toString())}
+              suffix={tokenA}
+            />
           )
         }
+        valueClassName={classNames(amount.gt(0) && 'text-primary-10')}
       />
-      {!isInitialDeposit && (
+      {amount.gt(0) && Number(newPoolBalanceB) > 0 && (
+        <SimpleTableRow
+          label=""
+          value={
+            <AmountRenderer
+              value={fromWei(decimalic(newPoolBalanceB).toString())}
+              suffix={BITCOIN}
+              precision={BTC_RENDER_PRECISION}
+            />
+          }
+          valueClassName={classNames(amount.gt(0) && 'text-primary-10')}
```

Comment:
The same condition (`classNames(amount.gt(0) && 'text-primary-10')`) is used in 3 places, extract it to a variable and use instead.